ref-farming/tests/common/mod.rs

```diff
@@ -0,0 +1,245 @@
+use std::convert::TryFrom;
```

Comment:
this doesn't need to be a separate folder if it's just a single file. can be common.rs

Reply:
The reason to create a mod folder is that the testing would go over all files in tests fold, which cause an unnecessary testing output for this common file. While putting it in subfolder would avoid this.